internal/templates/new_index.html

```diff
@@ -11,19 +11,23 @@
 
 <script src="{{ url_for('static', filename='oboe-browser.js') }}"></script>
 <script src="{{ url_for('static', filename='tinycolor.js') }}"></script>
-    <script src="{{ url_for('static', filename='cartogram2.js') }}?v={{cartogram_version}}"></script>
+<script src="{{ url_for('static', filename='cartogram2.js') }}?v={{cartogram_version}}"></script>
+<script src="https://d3js.org/d3.v5.min.js"></script>
```

Comment:
The D3.JS script is already loaded in `base_layout.html`. Currently we are using D3 version 3. Do you need new features from version 5?